### ISSUE TITLE
 Learn Basic CSS by Building a Cafe Menu - Step 64

### ISSUE THREAD

**Issue by ilenia-magoni** [freeCodeCamp/freeCodeCamp#56079]
### Describe the Issue

In step 64 you are asked to create an anchor element, the anchor element in step 65 has a `target` attribute, but there is no test for this in step 64.

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-basic-css-by-building-a-cafe-menu/step-64

### Your code

```html
      <footer>
        <p><a href="https://www.freecodecamp.org">Visit our website</a></p>
      </footer>


```

### Expected behavior

The `target` element in step 65 should not come as a surprise. Step 64 should instruct to add it.

### System

NA

**Comment by Sembauke** [comment]
Okay, let's make sure the user adds the `target="_blank"` in step 64 by adding the following sentence:

> Make sure that the link opens in a new tab by adding a `target` attribute with the value `_blank`.

After adding that to the description make sure to add a test that tests for a `target` attribute on the anchor element.

**Comment by Supravisor** [comment]
In step 64, an anchor element is required in a paragraph element on line 54.
![image](https://github.com/user-attachments/assets/77f87ec0-90f7-450d-94fa-235d989c9636)

In step 65 on line 56 the anchor element does not contain a `target` attribute.
![image](https://github.com/user-attachments/assets/4e904799-7862-4098-8c3a-25b20f924130)

However in step 65 there is a `target` attribute in the anchor element one line 9
![image](https://github.com/user-attachments/assets/11ad7b52-c1b4-44fd-b3d6-6397cec4b24a)

In summary, there are three anchor elements in the final Cat Photo App.
The first one has a `target` attribute, the other two do not.

The `footer > p > a` does not appear to need a `target` attribute or test.

**Comment by gikf** [comment]
Hey @Supravisor

that appear to be different project.
